RSSTimeFormatter/DTReplacer.cs

```diff
@@ -9,57 +9,58 @@ public class DTReplacer : MonoBehaviour
 		public static string GetUniqueStringFromUniqueNode(string name, string node)
 		{
 			var configs = GameDatabase.Instance.GetConfigs(node);
-			if (configs.Length > 1) {
-				Debug.LogError(
-					"Multiple `" + node + "` configurations, falling back to default");
-			}
-			else if (configs.Length == 1) {
-				ConfigNode config = configs[0].config;
-				var formats = config.GetValues(name);
-				if (formats.Length > 1) {
-					Debug.LogError(
-						"`" + node + "` configuration has multiple `" + name + "` entries, falling back to default");
-				}
-				else if (formats.Length == 1) {
-					return formats[0];
-				}
-			}
-			return null;

```

Comment:
Because we're only interested in if the config and formats length are 1. We can reduce the number of if-checks, if we reverse the order and just check if length == 1 directly.